packages/media_core/src/endpoint.rs

```diff
@@ -113,6 +131,7 @@ pub enum EndpointReq {
     AudioMixer(EndpointAudioMixerReq),
     RemoteTrack(RemoteTrackId, EndpointRemoteTrackReq),
     LocalTrack(LocalTrackId, EndpointLocalTrackReq),
+    MessageChannel(String, EndpointMessageChannelReq),
```

Comment:
Oh, I missed this. As I comment before, can you changed String to MessageChannelId struct same with LocalTrackId and RemoteTrackId?

Reply:
@giangndm I'd already changed it, can you check it again